### ISSUE TITLE
propagator configuration needs better shorthand

### ISSUE THREAD

**Issue by lizthegrey** [open-telemetry/opentelemetry-go#1265]
```
global.SetTextMapPropagator(otel.NewCompositeTextMapPropagator(propagators.TraceContext{}, propagators.Baggage{}))
```

is impossible for a new user to figure out on their own, and is a magic incantation we should give a mnemonic if we don't just make it the default.

**Comment by Aneurysm9** [comment]
Agreed.  Unfortunately, the spec [now requires no-op default propagators](https://github.com/open-telemetry/opentelemetry-specification/pull/930/files).  Perhaps we should have a function in the `propagators` package that returns a composite propagator with `TraceContext` and `Baggage`.

**Comment by lizthegrey** [comment]
agree, a composite out of box propagator would be helpful to have a util function for.

**Comment by seanschade** [comment]
I will give you my experience from a user's point of view. I implemented this yesterday in one of our shared libraries and found it to be quite intuitive. 

We plan to create one or more custom propagators, and `otel.NewCompositeTextMapPropagator(...)` is an easy way to set them when initializing the telemetry. I created a map to help parse the `OTEL_PROPAGATORS` environment variable and set the global propagators.

```
var SupportedTextMapPropagators = map[string]otel.TextMapPropagator{
		"baggage":      propagators.Baggage{},
		"tracecontext": propagators.TraceContext{},
		"b3":           b3.B3{},
		"jaeger":       jaeger.Jaeger{},
}
```
